build_projects/dotnet-host-build/build.ps1

```diff
@@ -8,7 +8,7 @@ param(
     [string]$Architecture="x64",
     [string]$TargetArch="",
     [string]$ToolsetDir="",
-    [string]$Framework="netcoreapp1.0",
+    [string]$Framework="netcoreapp1.1",
```

Comment:
I don't know the answer to those questions, but I'd rather just quickly fix the part that is messed up than rewire that whole part of the code.